Author: Kurt Roeckx <kurt@roeckx.be>
Description: Adds debian ntp server.

diff -Naurp openntpd.orig/ntpd.conf openntpd/ntpd.conf
--- openntpd.orig/ntpd.conf	2006-05-14 05:29:21.000000000 +0000
+++ openntpd/ntpd.conf	2008-01-15 17:08:14.000000000 +0000
@@ -9,6 +9,10 @@
 # sync to a single server
 #server ntp.example.org
 
-# use a random selection of 8 public stratum 2 servers
+# use a random selection of 4 public stratum 2 servers
 # see http://twiki.ntp.org/bin/view/Servers/NTPPoolServers
-servers pool.ntp.org
+# and http://www.pool.ntp.org/
+server 0.debian.pool.ntp.org
+server 1.debian.pool.ntp.org
+server 2.debian.pool.ntp.org
+server 3.debian.pool.ntp.org
